test/e2e/page-objects/pages/confirmations/prague/confirmation.ts

```diff
@@ -0,0 +1,58 @@
+import { Driver } from '../../../../webdriver/driver';
+
+class Eip7702AndSendCalls {
```

Comment:
All our current confirmation page objects `extends Confirmation` as they all extend use the same React components so share common functionality?

Reply:
thanks for the suggestion, adding the extend to `TransactionConfirmation`